step-functions/step-functions-package/src/main/java/step/functions/packages/FunctionPackageManager.java

```diff
@@ -210,7 +210,7 @@ private List<Function> deleteFunctions(FunctionPackage previousFunctionPackage)
 	private FunctionPackage addOrUpdateFunctionPackage(FunctionPackage previousFunctionPackage,
 			FunctionPackage newFunctionPackage) throws Exception {
 		String packageLocation = newFunctionPackage.getPackageLocation();
-		if (packageLocation == null || packageLocation.trim().length() == 0) {

```

Comment:
@david-stephan I cannot see your comment

Reply:
![image](https://user-images.githubusercontent.com/19704350/218419370-c6803b82-defc-4207-b80c-93b3ba98c76c.png)

Reply:
ok, I see the problem and added back a part of the refactoring that was revert by mistake